### PR TITLE
monitoring: Add Loki and Flux logs to guide

### PR DESCRIPTION
**Pull request by stefanprodan**
Changes:
- add instructions on how to install Grafana Loki stack
- add Flux logs dashboard
- extend the Flux Grafana annotations docs
- add a section about structured logging

### CODE REVIEW

File: content/en/docs/guides/monitoring.md [hunk 1]
```diff
@@ -143,3 +197,51 @@ groups:
         annotations:
           summary: '{{ $labels.kind }} {{ $labels.exported_namespace }}/{{ $labels.name }} reconciliation has been failing for more than ten minutes.'
 ```
+
+## Logs
+
+The Flux controllers follow the Kubernetes structured logging conventions.
+The logs are written to `stderr` in JSON format, with the following common tags:
```
Comment (darkowlzz) [inline content/en/docs/guides/monitoring.md:204]: maybe
```suggestion
The logs are written to `stdout` and `stderr` in JSON format, with the following common tags:
```

Comment (stefanprodan) [reply content/en/docs/guides/monitoring.md:204]: We don't write the stdout

Comment (darkowlzz) [reply content/en/docs/guides/monitoring.md:204]: Sounds like I've some misunderstanding.

**Review by darkowlzz — APPROVED** [review]
LGTM!

**Pull request merged**